internlm/utils/model_checkpoint.py

```diff
@@ -283,19 +316,37 @@ def __init__(self, ckpt_config, model, model_config=None, model_config_file=None
             lr_scheduler (object): lr_scheduler obj.
             model_config (dict): model config.
         """
-        self.enable_save_ckpt = ckpt_config.enable_save_ckpt
-        self.checkpoint_every = ckpt_config.checkpoint_every
-        self.save_ckpt_folder = ckpt_config.save_ckpt_folder
-        self.snapshot_ckpt_folder = ckpt_config.snapshot_ckpt_folder
-        self.oss_snapshot_freq: int = ckpt_config.oss_snapshot_freq
-        self.stop_file_path = ckpt_config.stop_file_path
-        self.load_model_only_folder = ckpt_config.load_model_only_folder
+        self.enable_save_ckpt = get_config_value(ckpt_config, "enable_save_ckpt", False)
+        self.checkpoint_every = get_config_value(ckpt_config, "checkpoint_every", 1)
+        self.save_ckpt_folder = get_config_value(ckpt_config, "save_ckpt_folder", None)
+        self.oss_snapshot_freq: int = get_config_value(ckpt_config, "oss_snapshot_freq", 1)
```

Comment:
Set the default value of `self.oss_snapshot_freq` to a larger num, e.g., 50. So does the default value of `self.checkpoint_every` above.